pset-iterators.js

```diff
@@ -115,6 +139,17 @@ console.log('Problem 4:')
 
 // Add your code below this line
 
+const presidentsBetween1850and1900 = presidents.filter((president)=>{
+  const presidentYear = parseInt(president.took_office.split("-")[0])
+  return presidentYear >= 1850 && presidentYear <= 1900
```

Comment:
Way to keep data types in mind by converting a string into an integer! And great use of navigating through the object and method chaining.